src/sprite_battler.cpp

```diff
@@ -264,19 +264,20 @@ bool Sprite_Battler::IsIdling() {
 }
 
 int Sprite_Battler::GetWidth() const {
-	if (animation) {
-		return animation->GetWidth();
-	}
+	if (animation) return 48;
 	return Sprite::GetWidth();
 }
 
 int Sprite_Battler::GetHeight() const {
-	if (animation) {
-		return animation->GetHeight();
-	}
+	if (animation) return 48;
 	return Sprite::GetHeight();
 }
 
+bool Sprite_Battler::IsUsingAnimationAsSprite() {
+	if (animation) return true;
```

Comment:
You can shorten this while function to `return animation != nullptr;`

Reply:
Did the requested change.